readme.md

```diff
@@ -499,6 +499,19 @@ An instance of [`CacheableLookup`](https://github.com/szmarczak/cacheable-lookup
 **Note:** This should stay disabled when making requests to internal hostnames such as `localhost`, `database.local` etc.\
 `CacheableLookup` uses `dns.resolver4(..)` and `dns.resolver6(...)` under the hood and fall backs to `dns.lookup(...)` when the first two fail, which may lead to additional delay.
 
+###### dnsIpVersion
+
+Type: `'auto' | 'ipv4' | 'ipv6'`\
+Default: `'auto'`
+
+Indicates which DNS record family to use.\
+Values:
+ - `auto`: IPv4 (if present) or IPv6
+ - `ipv4`: only IPv4
+ - `ipv6`: only IPv6
+
+Note: if you are using the undocumented option `family`, `dnsIpVersion` will override it.
```

Comment:
Is there a deprecation warning?

Reply:
> Is there a deprecation warning?

I'm going to add it when we're fine with `deprecation-warining.ts` from #1255